tests/integration/steps_downtime.go

```diff
@@ -1,5 +1,7 @@
 package main
 
+import "time"
+
 // stepsDowntime tests validator jailing and slashing.
 //
 // Note: These steps are not affected by slash packet throttling since
```

Comment:
We'll have to evaluate if this comment is still true

Reply:
It's true for happy path steps